sysid-library/src/main/cpp/generation/SysIdSetup.cpp

```diff
@@ -266,20 +266,20 @@ void SetupEncoders(
 }
 
 static frc::SPI::Port GetSPIPort(std::string_view name) {
-  if (name == "kOnboardCS0") {
-    fmt::print("Setup SPI kOnboardCS0");
+  if (name == "SPI (Onboard CS0)") {
+    fmt::print("Setup SPI (Onboard CS0)\n");
     return frc::SPI::Port::kOnboardCS0;
-  } else if (name == "kOnboardCS1") {
-    fmt::print("Setup SPI kOnboardCS1");
+  } else if (name == "SPI (Onboard CS1)") {
+    fmt::print("Setup SPI (Onboard CS1)\n");
     return frc::SPI::Port::kOnboardCS1;
-  } else if (name == "kOnboardCS2") {
-    fmt::print("Setup SPI kOnboardCS2");
+  } else if (name == "SPI (Onboard CS2)") {
+    fmt::print("Setup SPI (Onboard CS2)\n");
     return frc::SPI::Port::kOnboardCS2;
-  } else if (name == "kOnboardCS3") {
-    fmt::print("Setup SPI kOnboardCS3");
+  } else if (name == "SPI (Onboard CS3)") {
+    fmt::print("Setup SPI (Onboard CS3)\n");
     return frc::SPI::Port::kOnboardCS3;
-  } else if (name == "SPI.kMXP") {
-    fmt::print("Setup SPI SPI.kMXP");
+  } else if (name == "SPI (MXP)") {
+    fmt::print("Setup SPI (MXP)\n");
```

Comment:
Either we make a separate integration test clause for this or we make it match the default format